src/openvic-simulation/country/CountryInstance.hpp

```diff
@@ -98,6 +104,8 @@ namespace OpenVic {
 
 		/* Budget */
 		fixed_point_t PROPERTY(cash_stockpile);
+		IndexedMap<Strata, StandardSliderValue> PROPERTY(tax_rate_by_strata);
```

Comment:
How/where are the min and max values adjusted based on modifiers (policy) or is that out of scope?
(Yes I know I wrote the issue and should've made the scope clearer....)

Reply:
Out of scope then, great follow up issue.